### DISCUSSION TITLE
OIDC working example

### DISCUSSION THREAD

**Discussion by k-p-ani** [authelia/authelia#5136] [Q&A]
Hello,
   I want to know whether is there any working example of which shows end to end working of OIDC, that is, sample application, which shows how OIDC configuration is done on client side and Authelia and how end to end the integration is working

Thanks
Ani

**Comment by james-d-elliott** [comment]
Several client configuration examples exist such as [this one for Gitea](https://www.authelia.com/integration/openid-connect/gitea/). There are several others there which showcase the individual applications and how the specific settings marry up. This supplements the primary [OpenID Connect 1.0 Configuration Guide](https://www.authelia.com/configuration/identity-providers/open-id-connect/).


Finally there is the [configuration template](https://github.com/authelia/authelia/blob/v4.37.5/config.template.yml) which showcases the entire configuration and the [Get Started Guide](https://www.authelia.com/integration/prologue/get-started/) which is intended as guide showing users the individual steps and considerations for a general setup.

**Reply by k-p-ani** [reply]
Hi James,
   Thanks for quick reply. I am looking for example on how to setup Authelia as OIDC sever. And then corresponding configuration on any web application where user gets authenticated and authorised by Authelia OIDC. 

I tried configuring Authelia OIDC configuration as per below link.
[https://www.authelia.com/configuration/identity-providers/open-id-connect/](url)

but what I observed is that, after successful authentication, Authelia instead of redirecting to uri mentioned in _configuration.yaml_ file under the key "**redirect_uris**", Authelia is redirecting to uri mentioned in the key "**default_redirection_url**".

Thanks in Advance
